cmd/generate/config/rules/rule.go

```diff
@@ -25,7 +25,7 @@ const (
 	// \x60 = `
 	secretPrefixUnique = `\b(`
 	secretPrefix       = `(?:'|\"|\s|=|\x60){0,5}(`
-	secretSuffix       = `)(?:['|\"|\n|\r|\s|\x60]|$)`
+	secretSuffix       = `)(?:['|\"|\n|\r|\s|\x60|;]|$)`
```

Comment:
NOTE: Added the semicolon character to the secret suffix. It looks like this updated every single rule in gitleaks.toml.